applications/nrf5340_audio/src/bluetooth/bt_renderer/volume/bt_rend_vol.c

```diff
@@ -244,9 +244,8 @@ int bt_rend_vol_down(void)
 						i, ret);
 				}
 			}
-
-			return 0;
 		}
+		return 0;
```

Comment:
@alexsven we need to discuss.

Reply:
Nothing to discuss here, I think. This looked like a bug, meaning it would only set the volume for the first device